src/actions/channels.ts

```diff
@@ -495,8 +496,9 @@ export function fetchMyChannelsAndMembers(teamId: string): ActionFunc {
 
         let channels;
         let channelMembers;
+        const shouldFetchArchived = (isMinimumServerVersion(getServerVersion(getState()), 5, 21));
```

Comment:
` getState()` is being called three times and `getServerVersion` to times in this action, are we expecting different result values for each time we call them? if not I would suggest using a variable to store the value returned by the function and then use the variable where needed.

Reply:
Changed to use a variable :)